voice/voice-ai/x/SpeechRecognition/SpeechRecognition.swift

```diff
@@ -162,53 +164,89 @@ class SpeechRecognition: NSObject {
     
     // MARK: - Sentence Handling
     
-    func handleEndOfSentence(_ recognizedText: String) {
-        // Add your logic here for actions to be performed at the end of the user's sentence.
-        // For example, you can handle UI updates or other necessary tasks.
-        // ...
-        print("handleEndOfSentence -- method called")
-        
-        self.isRequestingOpenAI = true;
-        self._isPaused = false;
-        
-        recognitionTask?.finish()
-        recognitionTask?.cancel()
-        recognitionTask = nil
-        recognitionRequest?.endAudio()
-        audioEngine.stop()
-        isRandomFacts = false
-        self.audioPlayer.playSound()
-        VibrationManager.startVibration()
-        if (self.conversation.count == 0) {
-            self.conversation.append(openAI.setConversationContext())
```

Comment:
the context is not appended in the new code

Reply:
technically, it is "prepended" - at the beginning of the prompt, before anything else